kevm-pyk/pyproject.toml

```diff
@@ -31,6 +31,7 @@ pytest-xdist = "*"
 
 [tool.poetry.scripts]
 kevm-pyk = "kevm_pyk.__main__:main"
+gst-to-kore = "kevm_pyk.gst_to_kore:main"
```

Comment:
Initializing unrelated imports and the argument parser in `__main__` has a significant overhead so I extracted the command to the `gst_to_kore` module.